src/components/share-overlay/share-overlay.js

```diff
@@ -179,175 +252,48 @@ class ShareOverlay extends BaseComponent {
   renderMainState(): React$Element<any> {
     return (
       <div className={this.state.state === shareOverlayState.Main ? 'overlay-screen active' : 'overlay-screen'}>
```

Comment:
this.state.state?

Reply:
changed semantics to `view` and `shareOverlayView`